src/ontology/mondo-edit.obo

```diff
@@ -480062,25 +480061,22 @@ is_a: MONDO:0019701 {source="Orphanet:99645", source="icd11.foundation:206703263
 
 [Term]
 id: MONDO:0020474
-name: cheirospondyloenchondromatosis
-def: "Cheirospondyloenchondromatosis is an extremely rare type of enchondromatosis of very early onset (from neonatal period to infancy) characterized by symmetrical multiple enchondromas with metacarpal and phalangeal involvement resulting in short hands and feet, platyspondyly, mild to moderate short stature and intellectual disability." [Orphanet:99647]
+name: obsolete cheirospondyloenchondromatosis
+def: "OBSOLETE. Cheirospondyloenchondromatosis is an extremely rare type of enchondromatosis of very early onset (from neonatal period to infancy) characterized by symmetrical multiple enchondromas with metacarpal and phalangeal involvement resulting in short hands and feet, platyspondyly, mild to moderate short stature and intellectual disability." [Orphanet:99647]
 comment: Reason of obsoletion: out of scope - MONDO:excludeHistoricalDisease. Term to consider: -
-subset: gard_rare {source="GARD:19682", source="MONDO:GARD"}
-subset: nord_rare {source="MONDO:NORD"}
-subset: obsoletion_candidate
-subset: rare
 synonym: "generalised enchondromatosis with platyspondyly" EXACT OMO:0003005 []
 synonym: "generalized enchondromatosis with platyspondyly" EXACT []
 xref: GARD:19682 {source="MONDO:GARD"}
-xref: icd11.foundation:1707937102 {source="MONDO:equivalentTo"}
-xref: MEDGEN:1390387 {source="MONDO:equivalentTo", source="MONDO:MEDGEN"}
-xref: Orphanet:99647 {source="MONDO:equivalentObsolete"}
-xref: SCTID:725104005 {source="MONDO:equivalentTo"}
-xref: UMLS:C4510810 {source="MEDGEN:1390387", source="MONDO:equivalentTo", source="MONDO:MEDGEN"}
-is_a: MONDO:0018230 {source="Orphanet:99647"} ! skeletal dysplasia
+xref: icd11.foundation:1707937102 {source="MONDO:obsoleteEquivalent"}
+xref: MEDGEN:1390387 {source="MONDO:obsoleteEquivalent", source="MONDO:MEDGEN"}
+xref: Orphanet:99647 {source="MONDO:obsoleteEquivalent"}
+xref: SCTID:725104005 {source="MONDO:obsoleteEquivalent"}
+xref: UMLS:C4510810 {source="MEDGEN:1390387", source="MONDO:obsoleteEquivalent", source="MONDO:MEDGEN"}
 relationship: excluded_subClassOf MONDO:0019708 {source="Orphanet:99647", source="https://orcid.org/0000-0001-5208-3432"} ! obsolete primary bone dysplasia with disorganized development of skeletal components
+property_value: IAO:0000231 OMO:0001000 {source="MONDO:excludeHistoricalDisease"}
 property_value: IAO:0000233 "https://github.com/monarch-initiative/mondo/issues/9206" xsd:anyURI
-property_value: IAO:0006012 "2025-09-01" xsd:string
+is_obsolete: true
+replaced_by: MONDO:0013941
```

Comment:
"replaced by" is used when we merge terms. It means "the concepts are the same, so you if you used this term before, you can use the replaced_by ID without thinking. 
When we obsolete a term because it is a "historical disease", we say that "we thought this was a disease but it is actually not a disease. Therefore here, "consider" is more appropriate. (ie if someone used this term, they would have to check and see whether it would be appropriate to use MONDO:0013941.